docs/code/conds/api/cron.py

```diff
@@ -7,4 +7,3 @@ def do_minutely():
 @app.task(cron('*/2 12-18 * Oct Fri'))
 def do_complex():
     "Run at every 2nd minute past every hour from 12 through 18 on Friday in October."
-    ...
```

Comment:
I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).